docs/source/api/user-guide/programming-framework/supporting-elements.rst

```diff
@@ -287,6 +297,6 @@ Practically speaking, the smaller the ``d_out``, the tighter your analysis will
 You might find it surprising that metrics and measures are never actually evaluated!
 The framework does not evaluate these because it only needs to relate a user-provided input distance to another user-provided output distance.
 Even the user should not directly compute input and output distances:
-they are :ref:`solved-for <determining-accuracy>`, :ref:`bisected <parameter-search>`, or provided by the Context API.
+they are :ref:`solved-for <accuracy-user-guide>`, :ref:`bisected <parameter-search>`, or provided by the Context API.
```

Comment:
@mccalluc : does it make sense to link the "Context API" at the end of the sentence

Reply:
Sure: will do.